### PR TITLE
Solution react_todo_app

### PR DESCRIPTION
**Pull request by NazariiAlieksieiev**
[DEMO](https://nazariialieksieiev.github.io/react_todo-app/)

### CODE REVIEW

File: src/components/TodoFilter.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+              'filter__link',
+              { selected: status === TodoStatus.Active },
+            )
+          }
+        >
+          Active
+        </SearchLink>
+
+        <SearchLink
+          params={{ status: 'completed' }}
```
Comment (Viktor-Kost) [inline src/components/TodoFilter.tsx:53]: Replace the hardcoded string 'completed' with the constant `TodoStatus.Completed` for consistency.

File: src/components/SearchLink.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  ...props
+}) => {
+  const [searchParams] = useSearchParams();
+
+  return (
+    <Link
+      to={{
+        search: getSearchWith(searchParams, params),
+      }}
+      {...props} // copy all the other props
```
Comment (Viktor-Kost) [inline src/components/SearchLink.tsx:20]: this comment is redundant

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Great job!
Some minor recommendations left to improve your code

**Review by sTorba24 — APPROVED** [review]
Good work!

**Pull request closed without merging**